geomesa-gs-security/src/main/scala/org/geomesa/gs/security/audit/ElasticRequestDataListener.scala

```diff
@@ -0,0 +1,127 @@
+/***********************************************************************
+ * Copyright (c) 2013-2020 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the GNU GENERAL PUBLIC LICENSE,
+ * Version 2 which accompanies this distribution and is available at
+ * https://opensource.org/licenses/GPL-2.0.
+ ***********************************************************************/
+
+package org.geomesa.gs.security.audit
+
+import java.lang.reflect.Type
+import java.net.{MalformedURLException, URL}
+import java.util.Date
+
+import com.google.gson._
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.http.HttpHost
+import org.elasticsearch.action.ActionListener
+import org.elasticsearch.client.{RequestOptions, RestClient, RestHighLevelClient}
+import org.elasticsearch.action.index.{IndexRequest, IndexResponse}
+import org.elasticsearch.common.xcontent.XContentType
+import org.elasticsearch.action.DocWriteResponse
+import org.geoserver.monitor.{RequestData, RequestDataListener}
+import org.opengis.geometry.BoundingBox
+
+
+
+class ElasticRequestDataListener extends RequestDataListener with LazyLogging {
+  import ElasticRequestDataListener.gson
+  val envVar = sys.env.getOrElse("ELASTICSEARCH_HOST", null)
```

Comment:
one thing that just came up with activities is that you might want/need to configure multiple hosts for the elastic client. it's probably worthwhile to split this var on some unlikely character combo, like maybe `,` (urls wouldn't have commas, right?) to allow the ability to specify multiple hosts.